### PR TITLE
Ignore extra files

### PR DESCRIPTION
**Pull request by DalenW**
This is an additional flag `--skip-extras` to skip the extra files like `PICT0036-edited.jpg ` that are pointless in my opinion. So far it covers `["-edited", "-effects"]`. I also recommend adding `(1)` to that list, but I figure that's pretty risky so I left it out. 

Also I accidentally branched this off the GeoData branch so don't merge this until that's merged.

### CODE REVIEW

**Comment by DalenW** [comment]
Just added 74199e6, which adds another option called `--skip-extras-harder`. This will ignore files like `PICT0003(5).jpg`, but only when `PICT0003.jpg`. This fixes my earlier concern. Is a little bit dangerous, but it's suited to my needs so if anyone else wants to use it they can. 

`--skip-extras-harder` will also do `--skip-extras`

**Comment by TheLastGimbus** [comment]
> So far it covers ["-edited", "-effects"].

Sooo - I also thought about this, and the reason I left is is because in my case, they are named in Polish, so `-edytowane` - and it's probably named differently in *every single language*. Thanks Google :+1:

I'm also afraid that someone might have manually renamed their photos like that - but if we add this just as an optional "experimental" paramater, it will be fine :+1:

> I accidentally branched this off the GeoData

Great, we will have merge conflicts, since we are changing stuff there, but it's okay

File: google_photos_takeout_helper/__main__.py [hunk 1]
```diff
@@ -104,21 +116,29 @@ def for_all_files_recursive(
...
         what = _os.path.splitext(file.lower())[1]
         if what not in photo_formats:
             return False
+        # skips the extra photo file, like edited or effects. They're kinda useless.
+        if args.skip_extras or args.skip_extras_harder:  # if the file name includes something under the extra_formats, it skips it.
+            for extra in extra_formats:
+                if extra in file.lower():
+                    return False
+        if args.skip_extras_harder:
+            search = '\([0-9]\)\.'
```
Comment (TheLastGimbus) [inline google_photos_takeout_helper/__main__.py:129]: Any digit, including 13, 123 etc
```suggestion
            search = '\(\d+\)\.'
```

**Review by TheLastGimbus — COMMENTED** [review]
My suggestion for "`(1) `" regex

**Comment by DalenW** [comment]
> > So far it covers ["-edited", "-effects"].
> 
> Sooo - I also thought about this, and the reason I left is is because in my case, they are named in Polish, so `-edytowane` - and it's probably named differently in _every single language_. Thanks Google 👍

Oh damn, that sucks.  Might just leave it as experimental. If someone wants to add to it then so be it.

**Comment by DalenW** [comment]
Added the improved regex plus added more comments.

**Comment by TheLastGimbus** [comment]
Thank you for PR as always!

**Pull request merged**